ddtrace/contrib/django/patch.py

```diff
@@ -411,7 +411,21 @@ def traced_get_response(django, pin, func, instance, args, kwargs):
                     span.error = 1
                 span.set_tag("django.response.class", func_name(response))
                 if hasattr(response, "template_name"):
-                    utils.set_tag_array(span, "django.response.template", response.template_name)
+                    # template_name is a bit of a misnomer, as it could be any of:
```

Comment:
is there a link we can add to docs to support why this code is this way?

Reply:
It's not directly documented - I can link to the point in django's code where they talk about it though

Reply:
That is fine too!

If you can be sure to link to a commit hash directly rather than `master`, that way the link will always work.